sdk/documenttranslator/ai-document-translator-rest/samples/v1/javascript/README.md

```diff
@@ -0,0 +1,66 @@
+---
+page_type: sample
+languages:
+  - javascript
+products:
+  - azure
+  - azure-cognitive-services
+  - azure-document-translator
+urlFragment: ai-document-translator-javascript
```

Comment:
Let's set `disableDocsMs: true` in the sample config for now until this package can be published. It looks like we'll need to add some way to add a `-rest` here, otherwise a track 2 convenience layer version of ai-document-translator would collide with the same urlFragment here.